Bugzilla/User.pm

```diff
@@ -2877,6 +2877,43 @@ sub user_id_to_login {
   return $login || '';
 }
 
+sub validate_password {
```

Comment:
I loved the clarity that `assert_password_is_secure` and `assert_passwords_match` provides.

What do you feel about having a single function `assert_valid_password` that always throws?  Callers can use `eval` (or ideally `try/catch`) to handle errors if required (we're already doing that in one place).

Reply:
Sounds good to me. Will do that.